# Homework: The Normal Curve and $t$-Tests

Conduct the following homework problems in this notebook opening up new code blocks as needed. When you are done, you may take the D2L HW1 quiz with this document open.

## Exercise 1

The iris data set is built-in with R. The first line in the code cell shows the first few rows of the data set.

* Display a data summary of of the Sepal.Width variable
* Create a _horizontal_ boxplot of the Sepal.Width variable; put the title "Distribution of Sepal Width" on the plot  
* Create a histogram of the Sepal.Width variable
* Create a density plot of the Sepal.Width variable
* In the Markdown cell AFTER the following code cell, describe the distribution of the Sepal.Width variable

In [1]:
head(iris)

,Sepal.Length,Sepal.Width,Petal.Length,Petal.Width,Species
,<dbl>,<dbl>,<dbl>,<dbl>,<fct>
1,5.1,3.5,1.4,0.2,setosa
2,4.9,3.0,1.4,0.2,setosa
3,4.7,3.2,1.3,0.2,setosa
4,4.6,3.1,1.5,0.2,setosa
5,5.0,3.6,1.4,0.2,setosa
6,5.4,3.9,1.7,0.4,setosa


### Exercise 1, continued

b. Describe the distribution of the Sepal.Width in the code block below.

c. Do the same steps with the Petal.Width.

## Exercise 2

The boxplots of the Sepal.Width and Petal.Width variables in the iris data set should show that one of the distributions has outliers, while the other one does not. Use the information given in the data summary for each variable to show mathematically whether outliers exist according to the "1.5 IQR" rule. As part of your answer, be sure to identify explicitly the upper and lower cutoff for outliers for each variable.

Place your answer below. (Although it is not required, I will be favorably impressed if your mathematical work is shown using LaTeX.)

## Exercise 3

Suppose you are conducting a left-tailed $z$-test for 1 proportion. You obtain a z-statistic of -1.26. Use the appropriate R command(s) to find the p-value for this test.

## Exercise 4

A 30-month study was conducted among teenagers who had smoked (tobacco) cigarettes at least once.  A survey was administered to the participants to gauge the extent of the addiction they had developed to nicotine.  Scores on the survey range from 0 to 10, with higher numbers indicating a higher degree of addiction.  The summary statistics for the survey are given by biological sex. Do have evidence that among teens who have tried smoking, males and females differ in their level of nicotine dependence? If so, at what significance level?

<table style="width:65%">
  <tr>
    <th>Group</th>
    <th style="text-align: center;">Sample Size</th> 
    <th style="text-align: center;">Mean Score</th> 
    <th style="text-align: center;">Standard Deviation</th>
  </tr>
  <tr>
    <td><b>Females</b></td>
    <td style="text-align: center;">150</td>
    <td style="text-align: center;">2.8</td>
    <td style="text-align: center;">3.6</td>
  </tr>
  <tr>
    <td><b>Males</b></td>
    <td style="text-align: center;">182</td>
    <td style="text-align: center;">1.6</td>
    <td style="text-align: center;">2.9</td>
  </tr>
</table>

## Exercise 5

### Research Question: Do younger women (age < 20) use less aggressive humor than overall population average of 29? Test at the $\alpha = 0.5$ level.

We know two things about aggressive humor from prior research:

1. Biological females use less aggressive humor than males.
2. Human beings use less aggressive humor as they age.

Thus, the research question makes sense. You should consider the following steps:

- Work with the **personality** data frame.
- Keep only the variables Age and HSAG.
- Use subsetting techniques to retain only rows where the value in the Sex column is an “F”.
- Use subsetting techniques to retain only rows where the value in the Age column is less than 20.

## Exercise 6

### Research Question: Do Older College Consume More Caffeine and Get Less Sleep? Test at the $\alpha = 0.1$ level.

The grouping variable G21 has two levels, Y for those answering yes to whether their current age was 21 years old or older, and N for those answering no. We will perform 1-tailed $t$-tests on two variables: Sleep, the number of hours of sleep in last 48 including naps (divided by 2, to give an average per night), and Caff, the number of 12 ounce servings of caffeine in last 24 hours. Our hypotheses are that older students will get less sleep and consume more caffeine.

## Exercise 7

### Research Question: Do biological males at North Georgia use more coping humor than females? Test at the $\alpha = 0.05$ level.

Coping Humore (also known as gallows humor) describes the practice of using humor to cope when bad things happen in life. The variable in the **personality** data set for coping humore is **CHS**. You should consider the following steps:

- Work with the **personality** data frame.
- Keep only the variables Sex and CHS.
- Conduct all relevant phases of the hypothesis test indicated by the question.

## Exercise 8

Given the following normal distributions, answer the questions indicated below.

- IQ ~ N(100, 15)
- SAT ~ N(1000, 200)
- ACT ~ N(21,5)

### Questions

a. Find the 75th percentile IQ score.

b. Sally earns a 1290 on the SAT. What is her percentile score?

c. If Josie earns a 94th percentile score on the ACT, what is her raw score?

d. What is the percentile corresponding to an IQ of 81?